app/src/main/java/org/oppia/app/fragment/InjectableFragment.kt

```diff
@@ -16,8 +21,16 @@ abstract class InjectableFragment: Fragment() {
    */
   lateinit var fragmentComponent: FragmentComponent
 
-  override fun onAttach(context: Context?) {
+  override fun onAttach(context: Context) {
     super.onAttach(context)
     fragmentComponent = (requireActivity() as InjectableAppCompatActivity).createFragmentComponent(this)
   }
+
+  override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+    return super.onCreateView(inflater, container, savedInstanceState)
+  }
+
+  fun createViewComponent(view: View): ViewComponent {
```

Comment:
Ah, I forget why I added the view component. I think it was for the item selection view--it's a nice idea, but we don't need it right now. Removed the whole thing.

Reply:
Actually I made a mistake--this is used by SelectionInteractionView since it's using a new view scope. I also realized that I forgot to document this  in the top-level PR and will make that adjustment. The method appears used in Android Studio once the project is built.

Reply:
sounds good.